docs/smart-contracts/install/local-dev-node.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Local development network
+id: local-dev-node
+sidebar_position: 5
+description: "Start a local development network with a standalone blockchain node."
+---
+
+# Set up a development network
+
+Computers that run the Chainweb consensus protocol—the `chainweb-node` software—are a crucial component of the Kadena blockchain.
+Nodes provide the compute, storage, and network resources that enable the Kadena blockchain to be secure and decentralized.
+Nodes also provide the engine for transaction processing, data verification, and executing smart contract functions.
+Because of the role they play in receiving and processing transactions, it's important to include at least one standalone node as part of your development environment.
+
+In most cases, you can run a single node to establish a local development network that you can then use to test smart contract operations in an isolated and controlled environment before deploying to a public test or production network.
+
+There are two primary ways you can set up a local development network with the `chainweb-node` program:
+
+- You can download a complete local development network that includes the `chainweb-node` program, a local block explorer, the Pact interpreter that runs in a browser, and other tools. 
+  With this option, you can use a default set of tools and configuration settings from a Docker image to run the development network inside of a Docker container.
+  For instructions, see [Start the development network](#start-the-development-network).
+- You can download and run the `chainweb-node` program as a standalone release binary from its GitHub repository.
```

Comment:
I think officially we provide Docker images, that include the chainweb-node binary. @chessai is this correct?